src/main/java/seedu/foodrem/logic/commands/generalcommands/HelpCommand.java

```diff
@@ -11,28 +12,21 @@
  * Format full help instructions for every command for display.
  */
 public class HelpCommand extends Command {
-
-    public static final String COMMAND_WORD = CommandWord.HELP_COMMAND.getValue();
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Displays help for FoodRem.\n"
-            + "Example: " + COMMAND_WORD;
-    public static final String SHOWING_HELP_MESSAGE = "Opened help window.";
     public static final String DEFAULT_HELP_MESSAGE = "Please refer to the user guide.";
 
     // To be displayed in a new help window
-    // TODO: Update this to match user guide
-    public static final String USER_GUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
-    public static final String MORE_INFORMATION = "For more information please head to:\n" + USER_GUIDE_URL;
-
-    public static final String HELP_FOR_ALL_COMMANDS =
+    public static final String USER_GUIDE_URL = "https://ay2223s1-cs2103t-w16-2.github.io/tp/UserGuide";
+    private static final String MORE_INFORMATION = "For more information please head to:\n" + USER_GUIDE_URL;
+    private static final String COMMAND_WORD = HELP_COMMAND.getCommandWord();
+    private static final String HELP_FORMAT_GENERAL =
```

Comment:
Can I clarify why `HELP_FORMAT_GENERAL` and `HELP_FORMAT_SPECIFIC` is not defined in `CommandType.java`, as with all other help messages? I'm also a bit unclear on the reason how defining the usage for the Help Command here will interact with `getUsage()`.

Reply:
Good point, I will move both to `CommandType.java`. The usage for all commands should be defined in the enum for easy management of all the commands.